### ISSUE TITLE
VS integration test failure in CSharpExtractMethod.ExtractViaCodeAction

### ISSUE THREAD

**Issue by dpoeschl** [dotnet/roslyn#21175]
https://ci.dot.net/job/dotnet_roslyn/job/master/job/windows_release_vs-integration_prtest/6691/artifact/Binaries/Release/UnitTests/VisualStudioIntegrationTests/xUnitResults/Roslyn.VisualStudio.IntegrationTests.dll.out.log

```
    Roslyn.VisualStudio.IntegrationTests.CSharp.CSharpExtractMethod.ExtractViaCodeAction
      System.Exception : The following expected item(s) not found:
      Extract Method
      
      Stack Trace:
           at Microsoft.VisualStudio.IntegrationTest.Utilities.TestUtilities.ThrowIfExpectedItemNotFound[TCollection](IEnumerable`1 actual, IEnumerable`1 expected)
           at Microsoft.VisualStudio.IntegrationTest.Utilities.OutOfProcess.TextViewWindow_OutOfProc.Verifier`1.CodeActions(IEnumerable`1 expectedItems, String applyFix, Boolean verifyNotShowing, Boolean ensureExpectedItemsAreOrdered, Nullable`1 fixAllScope, Boolean blockUntilComplete)
        q:\roslyn\src\VisualStudio\IntegrationTest\IntegrationTests\CSharp\CSharpExtractMethod.cs(87,0): at Roslyn.VisualStudio.IntegrationTests.CSharp.CSharpExtractMethod.ExtractViaCodeAction()
```

**Comment by tmat** [comment]
Failing again:

https://dev.azure.com/dnceng/public/_build/results?buildId=654807&view=ms.vss-test-web.build-test-results-tab&runId=20330382&resultId=100267&paneView=debug
```
\r\nExpected:\r\n[310..319)\r\n[394..403)\r\nActual:\r\n[310..319)\r\nDifferences:\r\n [310..319)\r\n--> [394..403)\r\n\r\nExpected: True\r\nActual: False

at Roslyn.Test.Utilities.AssertEx.SetEqual[T](IEnumerable`1 expected, IEnumerable`1 actual, IEqualityComparer`1 comparer, String message, String itemSeparator, Func`2 itemInspector) in /_/src/Test/Utilities/Portable/Assert/AssertEx.cs:line 440
   at Roslyn.VisualStudio.IntegrationTests.VisualBasic.BasicExtractMethod.ExtractViaCodeAction() in /_/src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicExtractMethod.cs:line 95
```

**Comment by sharwell** [comment]
@tmat The new failure is unrelated and was fixed in #44487
